src/clients/python/__init__.py

```diff
@@ -266,6 +266,55 @@ def _raise_error(msg):
     raise ex
 
 
+def serialize_string_tensor(input_tensor):
+    """Serializes a string tensor into a flat numpy array of length prepend strings.
```

Comment:
Add description about the case of passing bytes directly?

Reply:
Done